CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# 0.101.1
+
```

Comment:
```suggestion
# 0.101.1

- On the user level, it's possible to set `default_parse_time_macros` to avoid specfile parsing issues like [packit/packit-service#2514](https://github.com/packit/packit-service/issues/2514). (#2408)
```

Reply:
I am not sure we want to include this, this was meant to be used for service config, not sure if there is a usecase to use it from user config (and not package config)

Reply:
OK, should I skip the release then?

@nforro can't people hit this locally as well?

Reply:
Sure they can, that's why `parse_time_macros` was introduced. Setting a default for it would only make sense if they wanted to have it defined for all projects.

Reply:
I suppose it could make sense for someone, so I'm not against it, but the changelog line is not accurate, the new config option doesn't prevent any issues, it can just make things more convenient.

Reply:
```suggestion
# 0.101.1

- On the user level, it's possible to set `default_parse_time_macros`, e.g. might be helpful in situations like [packit/packit-service#2514](https://github.com/packit/packit-service/issues/2514). (#2408)

```

Reply:
@nforro Is this better?